native/effektio/src/platform/desktop.rs

```diff
@@ -18,17 +23,48 @@ pub async fn new_client_config(base_path: String, home: String) -> Result<Client
     Ok(builder)
 }
 
-pub fn init_logging(filter: Option<String>) -> Result<()> {
+// this excludes macos, because macos and ios is very much alike in logging
+
+pub fn init_logging(log_dir: String, filter: Option<String>) -> Result<()> {
```

Comment:
we are still having thee almost identical copies of the same code here, don't we? please unify that into one fn that is called from within these...

Reply:
In fact, `init_logging()` has the specified implementation by platform, like `new_client_config()`.
So I split it into `android`, `ios` and `desktop`.
Will I unify `new_client_config()` too?

Reply:
How do these implementations actually differ? To me these look too similar to not unify them in one place. Same as for `new_client_config`, just reuse a `native` implementation:

```rust
pub async fn init_logging(log_dir: String, filter: Option<String) -> Result<()> {
    Ok(native::init_logging(log_dir, filter).await?)
}
```